packages/xod-fs/src/write.js

```diff
@@ -1,5 +1,5 @@
-
-import { curry } from 'ramda';
+import { curry, unless, test, concat, __ } from 'ramda';
```

Comment:
We import all Ramda functions in other places 🤔

Reply:
Yep, and at first I imported a whole Ramda, but this has led to a larger changeset, which is not relevant to the task (and there was an import of just `curry` function). So I decided to import some needed functions instead of the whole library.
If you think that it is better to import the entire Ramda — it's not a problem.

So, what do you think, @evgenykochetkov?